intro/mqtt/solution/src/main.rs

```diff
@@ -94,25 +90,18 @@ fn main() -> anyhow::Result<()> {
     }
 }
 
+// If we remove the hierarchy, we should not need to process the topic anymore
```

Comment:
Function comments should be `///`. I suggest starting with a one-line description of the function, then put more detail in a following paragraph. Like:

```
/// Handles one message received over MQTT.
///
/// More details go here. Blah blah blah. This
/// section might wrap.
```

Reply:
I did this and all comments changes (by mistake) in the mqtt_client_ai branch!